### PR TITLE
Use Utility function in InputLiteral component to determine the boolean values from JSON strings

### LINKED ISSUE #736: InputLiteral TypeError: _this.props.formData.items is undefined
When loading the `Sinopia:RT:Demo:SampleResourceFuller` or by expanding the Barcode in the spoofed `resourceTemplate:bf2:Monograph:Instance` resource template, a **TypeError: _this.props.formData.items is undefined** is being raised with a blank Editor view. This may be due to a possible issue with the defaults not being set currently in the Redux state for an `InputLiteral`

### PR DESCRIPTION
**Pull request by jgreben**
PR #738 Fixed #736. This PR is now only about using the Utility function to determine the boolean values from JSON strings.

### CODE REVIEW

**Comment by jgreben** [comment]
Waiting for #738 to be merged

**Comment by jcoyne** [comment]
@jgreben In which situations would the props have null formData?

**Comment by jgreben** [comment]
> @jgreben In which situations would the props have null formData?

@jcoyne OK, I see that `findNode` will set `formData` to `{}` by default. Thanks.

File: src/components/editor/property/InputLiteral.jsx [hunk 1]
```diff
@@ -118,13 +119,13 @@ export class InputLiteral extends Component {
     this.inputLiteralRef.current.focus()
   }
 
+  /**
+   * @return {bool} true if the field should be marked as required (e.g. not all obligations met)
```
Comment (ndushay) [inline src/components/editor/property/InputLiteral.jsx]: say default is false?  or is that obvious?

Comment (jcoyne) [reply src/components/editor/property/InputLiteral.jsx]: I think it's pretty clear from how it's documented that if it doesn't meet that condition it's false.

**Pull request merged**